boilerplate/tsconfig.json

```diff
@@ -14,7 +14,12 @@
     "target": "esnext",
     "lib": ["esnext", "dom"],
     "skipLibCheck": true,
-    "resolveJsonModule": true
+    "resolveJsonModule": true,
+    "paths": {
+      "app/*": ["app/*"],
+      "assets/*": ["assets/*"]
+    },
+    "baseUrl": "./"
```

Comment:
I thought maybe a baseUrl of "./" would be enough to achieve this? According to https://www.typescriptlang.org/tsconfig#baseUrl, looks like it should be.

Reply:
I will double check!

Reply:
Yeah you're right, pushing updates